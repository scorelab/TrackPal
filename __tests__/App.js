/**
 * @format
 * @lint-ignore-every XPLATJSCOPYRIGHT1
 */

import 'react-native';

jest.useFakeTimers();

describe('Authenticated Stack Navigator Test', () => {
  test('authenticatinng user is valid', () => {
    expect({ a: 5 }).toMatchSnapshot({
      a: 5
    }, 'snaphot 1');
  });

  test('bottm tab navigation is valid', () => {
    expect({ a: 4 }).toMatchSnapshot({
      a: 4
    }, 'snapshot 2');
  });
});