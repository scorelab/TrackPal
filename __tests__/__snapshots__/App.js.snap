// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Authenticated Stack Navigator Test authenticatinng user is valid: snaphot 1 1`] = `
Object {
  "a": 5,
}
`;

exports[`Authenticated Stack Navigator Test bottm tab navigation is valid: snapshot 2 1`] = `
Object {
  "a": 4,
}
`;
